llvm/lib/Target/X86/X86ISelLowering.cpp

```diff
@@ -23483,14 +23485,40 @@ static SDValue LowerVSETCC(SDValue Op, const X86Subtarget &Subtarget,
   SDLoc dl(Op);
 
   if (isFP) {
-    MVT EltVT = Op0.getSimpleValueType().getVectorElementType();
+    MVT SVT = Op0.getSimpleValueType();
```

Comment:
We usually use SVT for scalar VTs - maybe OpVT? The getSimpleValueType can also be merged with the isFP check above